README.md

```diff
@@ -60,33 +60,22 @@ $lb_components.each |$comp_name| {
   }
 }
 ```
-## Reference
-
-#### `data_capability`
 
-The data_capability type accepts a single parameter data which can be used to
-pass arbitrary data to other other nodes. There is no avilability check. See [examples/data.pp](https://github.com/puppetlabs/puppetlabs-app_modeling/blob/master/examples/data.pp)
-
-##### Parameters
-
-* `data`: Allows user to pass arbitrary data from from one component to another.
+## Reference
 
 ### Types
 
 #### `data_capability`
 
-This type passes information between nodes but has no availability check. It's
-useful if you need to pass data but don't care about whether services are
-actually running.
+The `data_capability` type accepts a single parameter that can be used to pass arbitrary data to other other nodes. There is no availability check. This is useful if you need to pass data between components but don't care if services are running. See [examples/data.pp](https://github.com/puppetlabs/puppetlabs-app_modeling/blob/master/examples/data.pp).
 
-##### `Parameters`
+##### Parameters
 
-* `data`: Arbitrary data to produce or consume usually this should be a hash.
+* `data`: Arbitrary data to produce or consume component information. This should be a hash.
```

Comment:
It's really up to users. I suggest a hash because to make it obvious why they don't need more than one param